CHANGES

```diff
@@ -23,6 +23,8 @@ Deprecated
   instead.
 * ``app.import_object()`` is deprecated.  Please use
   ``sphinx.util.import_object()`` instead.
+* Drop function based directive support.  For now, Sphinx only supports class
+  based directives.
```

Comment:
How about

    * Support for registering function-based directives with ``Application.add_directive`` is deprecated.